calendar/experiments/calendar/parent/ext-calendar-provider.js

```diff
@@ -144,7 +144,8 @@ class ExtCalendar extends cal.provider.BaseClass {
     super.id = val;
     if (this.id && this.uri) {
       try {
-        this.capabilities = JSON.parse(super.getProperty("extensionCapabilities"));
+        this.capabilities = JSON.parse(super.getProperty("extensionCapabilities"))|| 
+          this.extension.manifest.calendar_provider.capabilities || {};
```

Comment:
I see a little duplication between the try and the catch block. Could you pull the fallback out of the try catch to avoid the duplication?